Packs/Gatewatcher/Integrations/GCenterv102/GCenterv102.yml

```
@@ -0,0 +1,1119 @@
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.engines_report.12.scan_result
+      description: Analysis Result
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.engines_report.13.id
+      description: The Hash Pf The Engine
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.engines_report.13.threat_details
+      description: The Threat Found By The Engine
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.engines_report.13.scan_result
+      description: Analysis Result
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.engines_report.14.id
+      description: The Hash Pf The Engine
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.engines_report.14.threat_details
+      description: The Threat Found By The Engine
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.engines_report.14.scan_result
+      description: Analysis Result
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.engines_report.15.id
+      description: The Hash Pf The Engine
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.engines_report.15.threat_details
+      description: The Threat Found By The Engine
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.engines_report.15.scan_result
+      description: Analysis Result
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.magic_details
+      description: The File Magic
+      type: String
+    - contextPath: GCenter.Alerts.List.malware.total_found
+      description: The Malcore Number Of Engines That Found The File Suspicious / The Total Number Of Engines
+      type: String
+    - contextPath: GCenter.Alerts.List.sigflow.alert.action
+      description: Action
+      type: String
+    - contextPath: GCenter.Alerts.List.sigflow.alert.signature_id
+      description: Signature Id
+      type: String
+    - contextPath: GCenter.Alerts.List.sigflow.alert.gid
+      description: Gid
+      type: String
+    - contextPath: GCenter.Alerts.List.sigflow.alert.category
+      description: Category
+      type: String
+    - contextPath: GCenter.Alerts.List.sigflow.packet
+      description: Packet
+      type: String
+    - contextPath: GCenter.Alerts.List.sigflow.in_iface
+      description: In Which Interface The Alert Occurred
+      type: String
+    - contextPath: GCenter.Alerts.List.sigflow.stream
+      description: Is It Streaming (!= 0)
+      type: Number
+    - contextPath: GCenter.Alerts.List.sigflow.payload
+      description: Payload
+      type: String
+    - contextPath: GCenter.Alerts.List.sigflow.payload_printable
+      description: Payload Printable
+      type: String
+  - name: gw-get-alert
+    description: Get an alert by it's uid
+    arguments:
+    - name: uid
+      required: true
+      description: Alert identifier
+    outputs:
+    - contextPath: GCenter.Alerts.Single.sha256
```

Please change all the keys here to 'Alert' (in all the relevant contextPaths here).
Please update that also in the readme file.
```suggestion
    - contextPath: GCenter.Alert.Single.sha256
```